core/ui/src/main/java/com/yapp/core/ui/util/DateTime.kt

```
@@ -0,0 +1,60 @@
+
+import android.content.Context
+import com.yapp.core.ui.R
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+
+fun formatToDay(context: Context, date: String): String {
+    return try {
+        val parsedDate = LocalDate.parse(date)
+        context.getString(R.string.time_format_day, parsedDate.dayOfMonth)
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)